packages/babel-core/src/config/files/index-browser.ts

```diff
@@ -72,7 +72,7 @@ export function* resolveShowConfigPath(
   return null;
 }
 
-export const ROOT_CONFIG_FILENAMES = [];
+export const ROOT_CONFIG_FILENAMES: any[] = [];
```

Comment:
This is probably an array of strings?

Reply:
I don't know, it's an empty implementation file. But string[] should be fine.

Reply:
I don't know, it's an empty implementation file. But string[] should be fine.